buyer/issue/README.md

```diff
@@ -85,25 +85,27 @@ Response status codes:
 
 ## Order
 
-* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company permissions.
+* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company authorization.
 * `supplierAccountNumber`: the supplier account number as known in your ERP system
+
+{% hint style="warning" %}
+The `supplierAccountNumber` should be set on forehand in the Tradecloud connection with your supplier. You can set the account code when inviting a new connection or in the connection overview in the portal.
```

Comment:
A user can only add the account number when the connection request is accepted by the other party. This probably will change when we rethink the "new company / new connection" process. Atm I link the companies and add account number as extra service, as all connection requests are related to companies that do not yet exist on TC One.

Reply:
Changing to `The supplierAccountNumber should be set in the Tradecloud connection in the portal, after the connection request is accepted by the other party.`